### ISSUE TITLE
Wrong min PHP version 7.1 declared in wp-graphql 0.9.0

### ISSUE THREAD

**Issue by toriphes** [wp-graphql/wp-graphql#1349]
Hello!
The min php version declared for wp-graphql 0.9.0 is PHP-7.1 but the function `array_key_first` used in [TypeRegistry.php](https://github.com/wp-graphql/wp-graphql/blob/v0.9.0/src/Registry/TypeRegistry.php#L891) is available only from `PHP>=7.3`

Reference [PHP-DOC](https://www.php.net/manual/en/function.array-key-first.php)

**Comment by jasonbahl** [comment]
@toriphes oooh. Thanks for pointing this out.

We've been discussing what the minimum version of PHP should be for this project. 

WordPress is now officially recommending 7.4 as the minimum version, and even the official WordPress docker images only offer 7.3+ so our automated tests are only testing back to 7.3 right now. 

We'll need to either bump the version or change code like the `array_key_first` uses you pointed out (and update tests to run in 7.1 & 7.2)

**Comment by toriphes** [comment]
Hi @jasonbahl . Thanks for the answer!
PHP 7.3 has been marked to receive critical bug fixes only. 

If someone (like me) still use an older php version It might be useful to add a simple polyfill for `array_key_first`

```php
// wp-content/my-theme/functions.php
if (!function_exists('array_key_first')) {
    function array_key_first(array $array) { foreach ($array as $key => $value) { return $key; } }
}
```

**Comment by CodeProKid** [comment]
Pretty sure this was fixed in the 0.10.0 release https://github.com/wp-graphql/wp-graphql/releases/tag/v0.10.0 this can probably be closed.

**Comment by jasonbahl** [comment]
@CodeProKid you are correct. For some reason I thought there was also a mis-match in the `Requires PHP:` statement in the plugin docblock, but I think that was another issue resolved a while ago.
